troubadix/plugins/copyright_year.py

```diff
@@ -70,20 +70,20 @@ def check_lines(
                 and copyright_match.group(2) is not None
                 and not is_ignore_file(nasl_file, _IGNORE_FILES)
             ):
-                copyright_dict[line] = copyright_match.group(2)
+                copyrights.append((i, line, copyright_match.group(2)))
 
-        if not copyright_year:
+        if not creation_year:
             yield LinterError("Missing creation_date statement in VT")
 
         # key is the line where the copyright is found, value the year found
         # within that line
-        for key, value in copyright_dict.items():
-            if value != copyright_year:
-                report += f"\n{key.strip()}\n"
+        for nr, line, copyright_year in copyrights:
+            if copyright_year != creation_year:
+                report += f"Line {nr}: {line.strip()}\n"
 
         if len(report) > 0:
             yield LinterError(
                 "VT contains a Copyright year not matching "
-                f"the year in {copyright_year} at the following lines:\n"
+                f"the year in {creation_year} at the following lines:\n"
```

Comment:
IMHO we should print an Error for each found line individually here, like everywhere else ...
```python
        for nr, line, copyright_year in copyrights:
              if copyright_year != creation_year:
                  yield LinterError(
                     f"VT contains a Copyright year not matching"
                     f"the year in {creation_year} (line {nr})"
                  )
```

Reply:
Yes you are right. I'll change that.

Reply:
Done with https://github.com/greenbone/troubadix/pull/190/commits/18de06c062f609d33bc69798ac01ce3e54abfcbd